cookbook/AutoEval_Experiments.ipynb

```diff
@@ -0,0 +1,858 @@
+                        "    Found existing installation: pydantic 2.10.6\n",
+                        "    Uninstalling pydantic-2.10.6:\n",
+                        "      Successfully uninstalled pydantic-2.10.6\n",
+                        "  Attempting uninstall: pandas\n",
+                        "    Found existing installation: pandas 2.2.3\n",
+                        "    Uninstalling pandas-2.2.3:\n",
+                        "      Successfully uninstalled pandas-2.2.3\n",
+                        "  Attempting uninstall: httpx\n",
+                        "    Found existing installation: httpx 0.28.1\n",
+                        "    Uninstalling httpx-0.28.1:\n",
+                        "      Successfully uninstalled httpx-0.28.1\n",
+                        "  Attempting uninstall: lastmile\n",
+                        "    Found existing installation: lastmile 0.8.0\n",
+                        "    Uninstalling lastmile-0.8.0:\n",
+                        "      Successfully uninstalled lastmile-0.8.0\n",
+                        "Successfully installed annotated-types-0.7.0 anyio-4.8.0 certifi-2025.1.31 distro-1.9.0 h11-0.14.0 httpcore-1.0.7 httpx-0.28.1 idna-3.10 lastmile-0.8.0 numpy-2.2.2 pandas-2.2.3 pandas-stubs-2.2.3.241126 pydantic-2.10.6 pydantic-core-2.27.2 python-dateutil-2.9.0.post0 pytz-2025.1 six-1.17.0 sniffio-1.3.1 types-pytz-2025.1.0.20250204 typing-extensions-4.12.2 tzdata-2025.1\n",
+                        "Note: you may need to restart the kernel to use updated packages.\n"
+                    ]
+                }
+            ],
+            "source": [
+                "%pip install lastmile --upgrade"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## 1. Set Up AutoEval Client\n",
+                "\n",
+                "To interact with the LastMile AI API, set your API key as an environment variable. If you haven't already obtained an API key, please visit the LastMile AI dashboard."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 3,
+            "metadata": {},
+            "outputs": [
+                {
+                    "name": "stdout",
+                    "output_type": "stream",
+                    "text": [
+                        "✓ API key successfully configured!\n"
+                    ]
+                }
+            ],
+            "source": [
+                "import os\n",
+                "\n",
+                "api_token = os.environ.get(\"LASTMILE_API_TOKEN\")\n",
+                "\n",
+                "if not api_token:\n",
+                "    print(\"Error: Please set your API key in the environment variable LASTMILE_API_KEY\")\n",
+                "elif api_token == \"YOUR_API_KEY_HERE\":\n",
+                "    print(\"Error: Please replace 'YOUR_API_KEY_HERE' with your actual API key\")\n",
```

Comment:
Should this be LASTMILE_API_KEY? Error and value mismatch

```suggestion
                "api_key = os.environ.get(\"LASTMILE_API_KEY\")\n",
                "\n",
                "if not api_key:\n",
                "    print(\"Error: Please set your API key in the environment variable LASTMILE_API_KEY\")\n",
                "elif api_key == \"YOUR_API_KEY_HERE\":\n",
                "    print(\"Error: Please replace 'YOUR_API_KEY_HERE' with your actual API key\")\n",
```? The error variable name

Reply:
Either works but I think the message is inconsistent so I'll update